### DISCUSSION TITLE
Jedis version compatible with Redis 7.0.7

### DISCUSSION THREAD

**Discussion by nitinware** [xetorthio/jedis#3562] [Q&A]
- We are planning to migrate our AWS Elasticache Redis cluster from `5.0.6` to `7.0.7` to level new performance enhancements like **enhanced I/O multiplexing** and engine logs and slow logs supported above version `6.0`
- Currently we are using Jedis version `3.6.3` for connecting to Redis cluster from `5.0.6`
- Want to know which is the Jedis version suitable/compatible to work with Redis engine version`7.0.7`

**Comment by Amitgb14** [comment]
you don't need to upgrade jedis client, we had run 3.7.0 Jedis client on Redis 7.0.10 without noticed any issue
